### PR TITLE
Add Dockerfile and add Docker build/run instructions in README

### LINKED ISSUE #70: Dockerize backend
This would simplify deployment on the cloud platform

### PR DESCRIPTION
**Pull request by mikesir87**
Resolves #70

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,18 @@ Run: ```$ pip install -r requrements.txt```
...
+docker build -t yaptide .
+```
+
+Once built, you can run a container using the following command:
+
+```shell
+docker run -dp 5000:5000 yaptide
+```
+
+Once it's running, the app will be available at [http://localhost:5000](http://localhost:5000).
```
Comment (grzanka) [inline README.md]: Nice work, but can you also add an information how to stop the container ?

Comment (grzanka) [reply README.md]: Users less familiar with docker could be surprised that the process goes into background.

Comment (mikesir87) [reply README.md]: Ah! Gladly! 👍

File: README.md [hunk 2]
```diff
@@ -11,3 +11,25 @@ Run: ```$ pip install -r requrements.txt```
...
+
+When you're ready to stop the container, use the following command:
+
+```shell
+docker stop yaptide
+docker rm yaptide
+```
```
Comment (grzanka) [inline README.md:35]: This doesn't work for me:

```shell
root@test01:/home/ubuntu/yaptide# docker ps
CONTAINER ID   IMAGE     COMMAND   CREATED   STATUS    PORTS     NAMES
root@test01:/home/ubuntu/yaptide# docker run -dp 5000:5000 yaptide
2d20f6a01ef9e89961988a188e3060fa580c004fbc2a164bcaf113951b6d8f25
root@test01:/home/ubuntu/yaptide# docker ps
CONTAINER ID   IMAGE     COMMAND                  CREATED         STATUS         PORTS                                       NAMES
2d20f6a01ef9   yaptide   "flask run --host=0.…"   8 seconds ago   Up 6 seconds   0.0.0.0:5000->5000/tcp, :::5000->5000/tcp   naughty_robinson
root@test01:/home/ubuntu/yaptide# docker stop yaptide
Error response from daemon: No such container: yaptide
root@test01:/home/ubuntu/yaptide# docker rm yaptide
Error: No such container: yaptide
```

But if I use automatically generated name, the container is gone:
```shell
root@test01:/home/ubuntu/yaptide# docker ps
CONTAINER ID   IMAGE     COMMAND                  CREATED              STATUS          PORTS                                       NAMES
2d20f6a01ef9   yaptide   "flask run --host=0.…"   About a minute ago   Up 58 seconds   0.0.0.0:5000->5000/tcp, :::5000->5000/tcp   naughty_robinson
root@test01:/home/ubuntu/yaptide# docker stop naughty_robinson
naughty_robinson
root@test01:/home/ubuntu/yaptide# docker rm naughty_robinson
naughty_robinson
root@test01:/home/ubuntu/yaptide# docker ps
CONTAINER ID   IMAGE     COMMAND   CREATED   STATUS    PORTS     NAMES
```

```

File: README.md (hunk 1 above)
Comment (grzanka) [reply README.md]: See my comment below https://github.com/yaptide/yaptide/pull/71/files#r724226181

File: README.md (hunk 2 above)
Comment (mikesir87) [reply README.md:35]: Oh whoops! It's because I accidentally named the container locally, but didn't include that in the README. One more fix incoming.

Comment (grzanka) [reply README.md:35]: Now it works great !

**Review by grzanka — APPROVED** [review]
Looks good

**Pull request merged**